.eslintrc.json

```diff
@@ -1,235 +1,235 @@
 {
-    "extends": [
-        "airbnb",
-        "airbnb/hooks",
-        "plugin:import/recommended",
-        "plugin:import/typescript",
-        "prettier",
-        "plugin:@next/next/recommended",
-        "plugin:i18next/recommended",
-        "plugin:storybook/recommended"
-    ],
-    "parser": "@babel/eslint-parser",
-    "plugins": [
-        "prettier",
-        "react",
-        "react-func",
-        "sonarjs",
-        "unicorn",
-        "jsdoc"
-    ],
-    "rules": {
-        "i18next/no-literal-string": ["error", { "ignoreCallee": ["t"], "markupOnly": true, "onlyAttribute": [""], "validateTemplate": true, "ignore": ["—", "⇧", "⌥", "⌃", "©"] }],
-        "no-use-before-define": "off", // since we are using airbnb, it turns this on, we need to disabled it @see https://stackoverflow.com/a/64024916/1931451
-        "@typescript-eslint/no-use-before-define": "off",
-        "no-shadow": "off", // we need to disable this @see https://stackoverflow.com/a/63961972/1931451
-        "@typescript-eslint/no-shadow": ["error"],
-        "react/jsx-filename-extension": 0,
-        "react/jsx-props-no-spreading": 0,
-        "react/function-component-definition": 0,
-        "react/jsx-no-useless-fragment": 0,
-        "react/no-unstable-nested-components": 0,
-        "react/boolean-prop-naming": ["error", { "rule": "^(is|has|should)[A-Z]([A-Za-z0-9]?)+" }],
-        "react/destructuring-assignment": 2,
-        "react/no-children-prop": 2,
-        "react/no-deprecated": 2,
-        "react/no-multi-comp": 2,
-        "react/jsx-fragments": 2,
-        "react/jsx-handler-names": 2,
-        "react/jsx-sort-default-props": 2,
-        "react/jsx-key": 1,
-        "react-func/max-lines-per-function": ["error", {
-            "max": 30,
-            "skipBlankLines": true,
-            "skipComments": true
+  "extends": [
+      "airbnb",
+      "airbnb/hooks",
+      "plugin:import/recommended",
+      "plugin:import/typescript",
+      "prettier",
+      "plugin:@next/next/recommended",
+      "plugin:i18next/recommended",
+      "plugin:storybook/recommended"
+  ],
+  "parser": "@babel/eslint-parser",
+  "plugins": [
+      "prettier",
```

Comment:
Brother, please don't commit any styling changes in your PR. Thank you!

Reply:
@osamasayed, 

Assalamualaikum, 
can you please re-check again? 

Sorry for, my mistake.